sensors/social/sensorhub-driver-twitter/build.gradle

```diff
@@ -13,6 +13,13 @@ osgi {
   manifest {
     attributes('Bundle-Vendor': 'Botts Innovative Research Inc.')
     attributes('Bundle-Activator': 'org.sensorhub.impl.sensor.twitter.Activator')
+    attributes('Import-Package': '''
+      !dalvik.*, !java.*, !javax.crypto.*, !javax.imageio.*, !com.google.appengine.*,
+      !javax.net.*, !javax.naming.*, !javax.management.*, !javax.security.*, !javax.sql.*, !javax.swing.*,
+      !javax.transaction.*, !javax.xml.*, !com.sun.*, !org.apache.avalon.*, !org.apache.log.*,
+      !org.apache.log4j.*, !org.ietf.jgss.*,
+      org.osgi.service.repository;resolution:=optional,
```

Comment:
Don't think we need to import "org.osgi.service.repository;resolution:=optional"

Reply:
No need, Ok for building and installing:  44b5dcee197e0646698f4b3a72a60956535c1299